src/REVDeployer.sol

```diff
@@ -883,13 +889,29 @@ contract REVDeployer is ERC2771Context, IREVDeployer, IJBRulesetDataHook, IJBCas
         // (which will be 1 greater than the current count).
         if (originalRevnetId == 0) revnetId = PROJECTS.count() + 1;
 
-        // Deploy the tiered ERC-721 hook contract.
-        // slither-disable-next-line reentrancy-benign
-        hook = HOOK_DEPLOYER.deployHookFor({
-            projectId: revnetId,
-            deployTiersHookConfig: tiered721HookConfiguration.baseline721HookConfiguration,
-            salt: keccak256(abi.encode(tiered721HookConfiguration.salt, _msgSender()))
-        });
+        {
+            // Normalize and encode the configurations.
+            (JBRulesetConfig[] memory rulesetConfigurations, bytes32 encodedConfigurationHash) =
+                _makeRulesetConfigurations(configuration);
+
+            // Deploy the tiered ERC-721 hook contract.
+            // slither-disable-next-line reentrancy-benign
+            hook = HOOK_DEPLOYER.deployHookFor({
+                projectId: revnetId,
+                deployTiersHookConfig: tiered721HookConfiguration.baseline721HookConfiguration,
+                salt: keccak256(abi.encode(tiered721HookConfiguration.salt, encodedConfigurationHash, _msgSender()))
+            });
+
+            _deployRevnetFor({
```

Comment:
Q @mejango , is it safe to move up this call from the bottom of the function? Its needed because we need to scope the `rulesetConfigurations` and `encodedConfigurationHash` otherwise we get a `stack too deep`.

Reply:
i think safe yes.